ADMVEDU/Models/Constants/MediaType.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  MediaType.swift
+//  ADMVEDU
+//
+//  Created by Bobr, Andrey on 6.05.21.
+//
+
+import Foundation
+
+enum MediaType: String, CaseIterable {
+    case movie = "Movie"
```

Comment:
raw values look like titles, in this case that should be localized and returns not as raw values but as a separate custom property of enum